Xamarin.Forms.Maps.Android/MapRenderer.cs

```diff
@@ -51,6 +51,24 @@ public override SizeRequest GetDesiredSize(int widthConstraint, int heightConstr
 		{
 			return new SizeRequest(new Size(Context.ToPixels(40), Context.ToPixels(40)));
 		}
+		
+		public virtual void OnMapReady(GoogleMap map)
```

Comment:
In fact now that I think of it, we should leave the explicity implementation of this method, and then have that call the protected virtual method. This reduces the external API surface area while retaining the desired effect.